Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -1620,23 +1620,22 @@ protected String getTitleDefinition(String projectTitle, String docType) throws
     }
 
     /**
-     * Conditions:
-     * isDocType.equals("") && isNotDocType.equals("")
-     *     - nothing was specified
-     * isNotDocType.equals("") && StringUtils.containsIgnoreCase(isDocType, docType)
-     *     - only duty was specified
-     * isDocType.equals("") && !StringUtils.containsIgnoreCase(isNotDocType, docType)
-     *    - only may not was specified
-     * !isDocType.equals("") && !isNotDocType.equals("") && StringUtils.containsIgnoreCase(isDocType, docType)
-     *                 && !StringUtils.containsIgnoreCase(isNotDocType, docType)
-     *    - both were specified
+     * Conditions: isDocType.equals("") && isNotDocType.equals("") - nothing was
```

Comment:
Please undo this changes as now the comment is not readable any more.

Reply:
This is caused by the projects code formatter. Remember that Javadoc is HTML, not text, so line wrapping is meaningless and can freely be changed. The previous comment already looked like this, just you didn't notice:

![javadoc](https://user-images.githubusercontent.com/3040657/42271677-2c9584c6-7f84-11e8-9b66-7461eae04b28.png)

Reply:
No, answer is to write proper HTML